src/modules/navigation/FavoriteListItem.tsx

```diff
@@ -12,36 +12,40 @@ import { FileLink } from './components/FileLink'
 
 import { useFileLink } from '@/modules/navigation/hooks/useFileLink'
 import { isNextcloudShortcut } from '@/modules/nextcloud/helpers'
+import { isSharedDriveFolder } from '@/modules/shareddrives/helpers'
 
 interface FavoriteListItemProps {
   file: IOCozyFile
   clickState: [string, (value: string | undefined) => void]
 }
 
+const makeIcon = (file: IOCozyFile): string | React.ComponentType =>
+  isNextcloudShortcut(file) || isSharedDriveFolder(file)
+    ? FileTypeServerIcon
```

Comment:
So shared drives and Nextcloud shortcuts shared the same icon?

Reply:
Apparently yes